modules/mobile/spec/support/vcr_cassettes/appointments/VAOS_v2/get_appointments_with_mixed_provider_types.yml

```diff
@@ -0,0 +1,122 @@
+---
```

Comment:
I created this VCR by copying a few appointments from the ruben vcr and modifying them. Not ideal, but none of the appointments in that VCR fit some of these test cases.